src/main/java/seedu/toluist/commons/util/DateTimeUtil.java

```diff
@@ -0,0 +1,42 @@
+package seedu.toluist.commons.util;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+import java.util.List;
+
+import com.joestelmach.natty.DateGroup;
+import com.joestelmach.natty.Parser;
+
+/**
+ * Represent date nicely in human-readable format.
+ */
+public class DateTimeUtil {
+
+    public static LocalDateTime toDate(String stringDate) {
+        if (stringDate == null) {
+            return null;
+        } else {
+            Parser parser = new Parser();
+            List<DateGroup> dateGroups = parser.parse(stringDate);
+            if (dateGroups.isEmpty()) {
+                return null;
+            } else {
+                DateGroup dateGroup = dateGroups.get(0);
+                List<Date> dates = dateGroup.getDates();
+                if (dates.isEmpty()) {
+                    return null;
+                } else {
+                    Date date = dates.get(0);
+                    return LocalDateTime.ofInstant(date.toInstant(), ZoneId.systemDefault());
+                }
```

Comment:
Re-format the code to remove the nesting
e.g You can do
```
if (stringDate == null) {
            return null;
}

Parser parser = new Parser();
```
See how an indentation is eliminated w/o actually changing any logic